AnnoDesigner/Localization/Localization.cs

```diff
@@ -182,7 +182,19 @@ public static void Init(ICommons commons)
                         { "ViewLicenses" , "View open source Licenses" },
                         { "ExternalLinkConfirmationMessage" , "This will open a new tab in your default web browser. Continue?" },
                         { "ExternalLinkMessageTitle" , "Opening an external link" },
-                        { "RecentFiles" , "Recent Files" }
+                        { "RecentFiles" , "Recent Files" },
+                        { "UpdatePrefCurrentVersion" , "Current Version" },
```

Comment:
These key names don't need to be shortened - `UpdatePreferencesABC` is instantly understandable, whereas `UpdatePrefABC` is not (at least to me).

Reply:
Ok, Is done.